src/algebra/associated.lean

```diff
@@ -174,6 +188,17 @@ have hpd : p ∣ x * y, from ⟨z, by rwa [domain.mul_right_inj hp0] at h⟩,
   (λ ⟨d, hd⟩, or.inl ⟨d, by simp [*, pow_succ, mul_comm, mul_left_comm, mul_assoc]⟩)
   (λ ⟨d, hd⟩, or.inr ⟨d, by simp [*, pow_succ, mul_comm, mul_left_comm, mul_assoc]⟩)
 
+/-- If `p` and `q` are irreducible, then `p ∣ q` implies `q ∣ p`. -/
```

Comment:
An iff version of this could be useful? For rewriting.

Reply:
I assume you mean the implication between `p | q` and `q | r` should be an iff, like e4d6cf4ad8a46abd77d99a3305b067541975736e, right?